sklearn/metrics/classification.py

```diff
@@ -1502,6 +1502,11 @@ def classification_report(y_true, y_pred, labels=None, target_names=None,
         is also known as "sensitivity"; recall of the negative class is
         "specificity".
 
+        Show micro-average for multi-class classification is confusing,
```

Comment:
I would change the above, put the micro average last in the list and say it's only shown for multi-label or multi-class with a subset of classes because it's accuracy otherwise.

Reply:
done